docs/synapse/userguides/storm_ref_automation.rstorm

```diff
@@ -1135,6 +1136,9 @@ Users can interact with dmons using the Storm :ref:`storm-dmon` commands and the
 .. _Power-Ups: https://synapse.docs.vertex.link/en/latest/synapse/power_ups.html
 .. _Node Action: https://synapse.docs.vertex.link/projects/optic/en/latest/user_interface/userguides/custom_environ.html#create-a-custom-node-action
 .. _easy permissions: https://synapse.docs.vertex.link/en/latest/synapse/adminguide.html#easy-permissions
+.. _Node Actions: https://synapse.docs.vertex.link/projects/optic/en/latest/user_interface/userguides/custom_environ.html#create-a-custom-node-action
+.. _Storm Editor Tool: https://synapse.docs.vertex.link/projects/optic/en/latest/user_interface/userguides/storm_editor_tool.html
 .. _Synapse-AlienVault: https://synapse.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-alienvault/index.html
 .. _Synapse-Nettools: https://synapse.docs.vertex.link/projects/nettools/en/latest/
 .. _control flow: https://synapse.docs.vertex.link/en/latest/synapse/userguides/storm_adv_control.html
+.. _easy perms: https://synapse.docs.vertex.link/en/latest/synapse/adminguide.html#easy-permissions
```

Comment:
We have an existing link to this as `_easy permissions`, do we want to just use the same one in both places?